secretmanager/secretmanager_test.go

```diff
@@ -188,6 +191,26 @@ func TestCreateSecret(t *testing.T) {
 	}
 }
 
+func TestCreateSecretWithLabels(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	secretID := "createSecretWithLabels"
+	labelKey := "secretmanager"
+	labelValue := "rocks"
+
+	parent := fmt.Sprintf("projects/%s", tc.ProjectID)
+	defer testCleanupSecret(t, fmt.Sprintf("projects/%s/secrets/%s", tc.ProjectID, secretID))
```

Comment:
this defer should happen only *after* the secret has been successfully created, so move this down to 208.
As written, the cleanup will fail any time the create fails, because it cannot delete a nonexistent secret.